UPGRADE-3.2.md

```diff
@@ -69,12 +71,6 @@ Form
    }
    ```
 
-FrameworkBundle
----------------
-
- * The service `serializer.mapping.cache.doctrine.apc` is deprecated. APCu should now
-   be automatically used when available.
```

Comment:
What about moving the other items here? FrameworkBundle would then be sorted properly into the sections.